packages/knip/src/plugins/eleventy/index.ts

```diff
@@ -13,14 +16,31 @@ const isEnabled: IsPluginEnabledCallback = ({ dependencies }) => hasDependency(d
 
 const ENTRY_FILE_PATTERNS = ['.eleventy.js', 'eleventy.config.{js,cjs}'];
```

Comment:
The `ENTRY_FILE_PATTERNS` should become `CONFIG_FILE_PATTERNS`, otherwise the results don't end up in `findDependencies` - now I realize maybe this is why the tests using `main()` didn't work properly?

Reply:
That makes sooo much more sense!